packages/integration-tests/src/playbook/hardhat.config.ts

```diff
@@ -4,10 +4,16 @@ require("dotenv").config({ path: __dirname + "/.env" });
 import "@matterlabs/hardhat-zksync-deploy";
 import "@matterlabs/hardhat-zksync-solc";
 import "@nomiclabs/hardhat-ethers";
+
 import { localConfig } from "../config";
+import { Wallets } from "../entities";
 
 import type { HardhatUserConfig } from "hardhat/types";
 
+if (!process.env.WALLET_PRIVATE_KEY) {
```

Comment:
No need for this check and reassigning env vars is a bad pattern. I meant you can add a fallback in the place where it is used.
Like:

```
const wallet = new Wallet(process.env.WALLET_PRIVATE_KEY || Wallets.richWalletPrivateKey);
...
```

Reply:
Updated, thank you. @Romsters